cli/config_file.rs

```diff
@@ -286,14 +286,40 @@ pub struct LintFilesConfig {
 #[serde(default, deny_unknown_fields)]
 pub struct LintConfig {
   pub rules: LintRulesConfig,
-  pub files: LintFilesConfig,
+  pub files: FilesConfig,
+}
+
+#[derive(Clone, Copy, Debug, Deserialize)]
+#[serde(deny_unknown_fields, rename_all = "camelCase")]
+pub enum TextWrap {
+  Always,
+  Never,
+  Preserve,
+}
+
+#[derive(Clone, Debug, Default, Deserialize)]
+#[serde(default, deny_unknown_fields, rename_all = "camelCase")]
+pub struct FmtOptionsConfig {
+  pub use_tabs: Option<bool>,
+  pub line_width: Option<u32>,
+  pub indent_width: Option<u8>,
+  pub single_quote: Option<bool>,
+  pub text_wrap: Option<TextWrap>,
```

Comment:
The reason this is called `text_wrap` in dprint-plugin-markdown is because it controls how the `Text` node wraps. Since this is a top level config and not markdown specific, I think we should call this `prose_wrap` to align with prettier.

Reply:
I'm fine with changing name, but I'm unsure about that it's not Markdown specific - I haven't found a setting in `dprint-plugin-typescript` that would be relevant for this setting. Can you elaborate?